testing/asserts.ts

```diff
@@ -1,6 +1,9 @@
 // Copyright 2018-2023 the Deno authors. All rights reserved. MIT license.
 
-/** A library of assertion functions.
+/**
+ * @deprecated (will be removed after 0.200.0) Import from `std/asserts` instead.
```

Comment:
Maybe let's wait for a little more than usual deprecation period? `testing/asserts` are heavily depended by the ecosystem.

Reply:
The usual deprecation period is 3 minor versions. This is 9 minor versions, like the recent `std/semver` migration (#3385). Happy to increase this if you still think it should be pushed out.